utils/http_utils.go

```diff
@@ -14,6 +14,7 @@ type HTTPClient interface {
 
 var (
 	Client HTTPClient = &http.Client{}
+	GetSignature func(payload interface{}, workerId string) []byte = getSignature
```

Comment:
Could you explain the use of this variable? apparently it is only called once in a unit test.